javascript/modules/client/src/context.ts

```diff
@@ -6,7 +6,7 @@ import { UnsupportedProtocolError } from "@aragon/sdk-common";
 export { ContextParams } from "./internal/interfaces/context";
 
 const supportedProtocols = ["https:"];
-if(process.env.TESTING) {
+if(process && process.env && process.env.TESTING) {
```

Comment:
```suggestion
if(typeof process !== "undefined" && process.env?.TESTING) {
```

Reply:
The change you suggest only checks for undefined. if you don't use typeof it also checks for empty strings, 0 and NULL